libriscan/biblios/templates/biblios/components/forms/word_details.html

```diff
@@ -58,8 +58,8 @@ <h3 class="card-title">Word Details</h3>
       div.innerHTML = `
         <span class="font-medium">${suggestion.word}</span>
         <div class="flex items-center gap-3">
-          <span class="text-sm">${suggestion.confidence}%</span>
-          <progress class="${getProgressClass(suggestion.confidence)} w-24" value="${suggestion.confidence}" max="100"></progress>
+          <span class="text-sm">Frequency count: ${suggestion.confidence}</span>
+          <input type="range" min="0" max="100" value="${suggestion.confidence}" class="range w-24" disabled />
```

Comment:
Placeholder for Frequency. Replaced `progress` with `range` to show frequency for now.